bundle-workflow/src/test_workflow/integ_test/local_test_cluster.py

```diff
@@ -22,13 +22,21 @@ class LocalTestCluster(TestCluster):
     Represents an on-box test cluster. This class downloads a bundle (from a BundleManifest) and runs it as a background process.
     """
 
-    def __init__(self, work_dir, bundle_manifest, security_enabled, s3_bucket_name):
+    def __init__(self, work_dir, component_name, bundle_manifest, security_enabled, s3_bucket_name):
         self.manifest = bundle_manifest
         self.work_dir = os.path.join(work_dir, "local-test-cluster")
+        self.tmp_dir = os.path.join(self.work_dir, "tmp_dir")
         os.makedirs(self.work_dir, exist_ok=True)
+        os.makedirs(self.tmp_dir, exist_ok=True)
+        self.component_name = component_name
         self.security_enabled = security_enabled
         self.bucket_name = s3_bucket_name
         self.process = None
+        self.plugin_specific_config = {
+            "alerting": ["plugins.destination.host.deny_list: [10.0.0.0/8,  127.0.0.1]"],
+            "sql": ["script.context.field.max_compilations_rate: 1000/1m"],
+            "index-management": ["path.repo: %s" % os.path.realpath(self.tmp_dir)]
+        }
```

Comment:
This should go to test-manifest. Component owners should only have to modify in place and be done with it. 
Having config within the code brings in maintenance for us. This will eventually break when the plugin config changes.

Reply:
That's actually much better. Let me refactor it.

Reply:
Done please have a look. Thanks!